packages/destination-actions/src/destinations/fullstory/vars.ts

```diff
@@ -0,0 +1,140 @@
+import camelCase from 'lodash/camelCase'
+
+const isString = (x: any) => typeof x === 'string'
+
+const isBool = (x: any) => typeof x === 'boolean'
+
+const isReal = (x: any) => typeof x === 'number'
+
+const isInt = (x: any) => typeof x === 'number' && x - Math.floor(x) === 0
+
+const isDate = (x: any) => {
+  if (!x) {
+    return false
+  }
+  if (x.constructor === Date) {
+    return !isNaN(x as any)
+  }
```

Comment:
I omitted logic to check against `string` or `number` values since those comparisons will always be superseded by `isString` and `isReal` checks.